.github/workflows/lint-and-test.yml

```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install --trusted-host pypi.python.org -r requirements.txt
+          pip install uv
+          uv pip install -r pyproject.toml --system --all-extras
```

Comment:
I think if you use `uv sync .`, you can avoid the `pythonpath` shenanigans for pytest.

Reply:
That didn't work immediately, but it looks like `uv pip install -e . --system` did! Good enough for me

Reply:
Ah yes, I think because you want a system-wide installation. `uv sync` creates a virtual environment at `$PWD/.venv`.